src/test/java/com/epam/tc/hw2/ex1/ExerciseOneTest.java

```diff
@@ -0,0 +1,161 @@
+package com.epam.tc.hw2.ex1;
+
+import static org.assertj.core.api.AssertionsForClassTypes.assertThat;
+
+import com.epam.tc.hw2.AbstractChromeTest;
+import java.util.List;
+import org.assertj.core.api.SoftAssertions;
+import org.openqa.selenium.By;
+import org.openqa.selenium.NoSuchElementException;
+import org.openqa.selenium.WebElement;
+import org.openqa.selenium.support.ui.ExpectedConditions;
+import org.openqa.selenium.support.ui.WebDriverWait;
+import org.testng.annotations.Test;
+
+public class ExerciseOneTest extends AbstractChromeTest {
+
+    //Assert Browser title
+    @Test
+    public void checkTitle() {
+
+        assertThat(driver.getTitle()).as("Invalid Browser title").isEqualTo("Home Page");
+
+    }
+
+    //Perform login, Assert Username is loggined
+    @Test
+    public void login() {
+
+        try {
+            new WebDriverWait(driver, 5)
+                    .until(ExpectedConditions.presenceOfElementLocated(By.cssSelector("li.uui-profile-menu"))).click();
+            driver.findElement(By.id("name")).sendKeys("Roman");
+            driver.findElement(By.id("password")).sendKeys("Jdi1234");
+            driver.findElement(By.id("login-button")).click();
+        } catch (NoSuchElementException e) {
+            System.err.println("No element: " + e);
+        }
+
+        List<WebElement> searchResults = driver.findElements(By.id("user-name"));
+
+        SoftAssertions softly = new SoftAssertions();
+        softly.assertThat(searchResults.size() > 0).as("Login failed").isTrue();
+        softly.assertThat(driver.findElement(By.id("user-name")).getText()).isEqualTo("ROMAN IOVLEV");
+        softly.assertAll();
+
+    }
+
+    //Assert that there are 4 items on the header section are displayed and they have proper texts
+    @Test
+    public void navBarCheck() {
+
+        SoftAssertions softly = new SoftAssertions();
+        softly.assertThat(driver.findElement(By.cssSelector("ul.m-l8>li>a[href='index.html']")).isDisplayed());
```

Comment:
Could be simblified and maybe you don't need a css selector right here

Reply:
simplified, some css locators are replaced with By.linkText() ones